specification/informatica/resource-manager/Informatica.DataManagement/stable/2024-05-08/openapi.json

```diff
@@ -0,0 +1,2122 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Informatica.DataManagement",
+    "version": "2024-05-09",
```

Comment:
>"2024-05-09" [](http://example.com/codeflow?start=15&length=12)

Looks like this PR introduces new stable version of swagger, please create base commit for with either previous stable version or preview version(if it was arm reviewed earlier). Please follow: [https://eng.ms/docs/products/azure-developer-experience/design/api-specs/api-specs?tabs=openapi](https://eng.ms/docs/products/azure-developer-experience/design/api-specs/api-specs?tabs=openapi)

Reply:
This is our first version of the API specs (either private or stable) in this repo. Previous preview version is in the private repository: 

https://github.com/Azure/azure-rest-api-specs-pr/pull/16995
https://github.com/Azure/azure-rest-api-specs-pr/tree/RPSaaSMaster/specification/informatica

Reply:
You could still copy the files in the repo in your local branch and use the script to create a base of it.
Since you have chosen not done it, just FYI, it will take more time to complete the ARM review for this PR.